code/modules/antagonists/cult/cult_comms.dm

```diff
@@ -79,6 +79,100 @@
 			var/link = FOLLOW_LINK(M, user)
 			to_chat(M, "[link] [my_message]")
 
+///a button for requesting cultists come to your location, lasts for 3 minutes (during which time no one else can request assistance)
+/datum/action/innate/cult/request_aid
+	name = "Request Assistance"
+	desc = "Request a number of cultists of your choosing to your location."
+	button_icon_state = "assistance"
+	check_flags = AB_CHECK_HANDS_BLOCKED|AB_CHECK_IMMOBILE|AB_CHECK_CONSCIOUS
+	var/cooldown = 0
+	var/base_cooldown = 180 SECONDS
+
+/datum/action/innate/cult/request_aid/Activate()
+	var/datum/antagonist/cult/team = owner.mind.has_antag_datum(/datum/antagonist/cult,TRUE)
+	if(world.time < cooldown)
+		to_chat(owner, span_warning("You need to wait longer before requesting assistance again!"))
+		return
+	if(team.cult_team.need_of_help) //check to make sure no one else has already requested assistance
+		to_chat(owner, span_warning("Someone else is currently requesting assistance!"))
+		return
+	var/number = tgui_alert(owner, "How many cultists do you need?",, list("0", "1", "2", "3"))
+	if(number == "0")
+		to_chat(owner, "You decide not to request help.")
+		return
+	team.cult_team.need_of_help = owner
+	for(var/datum/mind/cultist as anything in get_antag_minds(/datum/antagonist/cult))
+		if(cultist.current && cultist.current.stat != DEAD && cultist.current.client)
+			var/area/location = get_area(owner)
+			var/title = "Acolyte"
+			if(owner.mind && owner.mind.has_antag_datum(/datum/antagonist/cult/master))
+				title = "Master"
+			else if(!ishuman(owner))
+				title = "Construct"
+			to_chat(cultist.current, span_cultitalic("<b>[title] [findtextEx(owner.name, owner.real_name) ? owner.name : "[owner.real_name] (as [owner.name])"] has requested the aid of [number] cultists in the [location.name]! <a href=?src=[REF(src)];pledge_assistance=1>(Pledge Assistance)</b>"))
```

Comment:
the [location.name] is bad, either use [location.name] or just [location] or \the [location.name]

Reply:
in the video for example you can see "requested assistance in the Engineering"